requirements.txt

```diff
@@ -1,2 +1,2 @@
-python-telegram-bot==20.6
+python-telegram-bot==13.5
```

Comment:
20.6 is the latest version of the framework https://docs.python-telegram-bot.org/en/v20.6/